README.md

```diff
@@ -13,35 +13,86 @@ method. For example, using [kind](https://kind.sigs.k8s.io/):
 kind create cluster --name xnat
 ```
 
-### Package the chart
+### Install the CNPG Operator
+
+We use the [CNPG Operator](https://github.com/cloudnative-pg/cloudnative-pg) to
+deploy Postgres. The operator can be installed using Helm:
+
```

Comment:
Just noting that this makes it seem as if this step is a requirement. But the PG operator chart is listed as a dependency so it would get installed when installing the xnat chart. Following this step would mean that the PG operator would get installed twice in different namespaces.

Reply:
I've just had a closer look at the CNPG Helm Chart docs and can now see why you've included this. Can we not list both the operator and cluster as dependencies in our chart? I think this would be better than having to handle the operator installation separately.

I don't think we'll want the installation to be available cluster-wide so we'd need to set  `config.clusterWide: false` and install in the specific XNAT namespace.

Reply:
I've now tested this and we can't install the operator and cluster together. So the operator installation will need to be done in our condenser cluster repo. In which case I guess it is OK to install the operator cluster-wide and have it watch for `Cluster` resources in all namespaces.

Reply:
> I've now tested this and we can't install the operator and cluster together.

sorry I should have made a note about that to save you some time! Yeah it would be nice if we could install both charts at the same time, but helm doesn't seem to be able to handle that